code/mission/missionparse.cpp

```diff
@@ -6177,40 +6177,44 @@ bool post_process_mission()
 
 	// before doing anything else, we must validate all of the sexpressions that were loaded into the mission.
 	// Loop through the Sexp_nodes array and send the top level functions to the check_sexp_syntax parser
+	// Cyborg -- If you are a ingame joiner, your sexps will be taken care of by the server, and checking will 
+	// actually crash mission load, since a list of objects present are also given to them only after this point.
+
+	if (!(Game_mode & GM_MULTIPLAYER) && (Net_player->flags & NETINFO_FLAG_INGAME_JOIN)) {
```

Comment:
This condition is incorrect (the logical NOT should apply to the conjunction of both flag checks)

Reply:
Thanks! Fixed